resources/themes/cura-light/theme.json

```diff
@@ -363,6 +363,12 @@
         "message_close_hover": [8, 7, 63, 255],
         "message_progressbar_background": [245, 245, 245, 255],
         "message_progressbar_control": [50, 130, 255, 255],
+        "message_success_icon_background": [36, 162, 73, 255],
+        "message_success_icon": [255, 255, 255, 255],
+        "message_warning_icon_background": [253, 209, 58, 255],
+        "message_warning_icon": [0, 0, 0, 255],
+        "message_error_icon_background": [218, 30, 40, 255],
+        "message_error_icon": [255, 255, 255, 255],
```

Comment:
(See the review in Uranium about the names and code duplication of these entries.)